HAB5/hab5.tex

```diff
@@ -0,0 +1,404 @@
+Among the tests are those for: camera alignment; camera stability; spectral response/quantum efficiency; software processing of simulations; payload NDVI/visual functioning while on ground; an electronics functional test with specific focus on power systems and APRS reliability; electronics environmental testing; integrated system testing.
+Time-stamping of information received by different processes may be considered. 
+It is also desired that metrics be included for post-flight analysis.
+
+Among the physical requirements for the payload, it is required that the HAB vehicle and payload system be better capable of protecting from environmental factors such as humidity, winds and temperature. Specific to this payload, the alignment of cameras is a question needing further exploration. An angle and separation optimal for the combination of data acquisition and securedness is required. Vibration or displacement during flights can significantly impair or corrupt all information gathered. Fixing this hurdle is critical for future projects which may look to infer distance data with a combination of two cameras or a radar altimeter setup. While the project doesn't expect that these will all be met as results, it is firmly held that the steps taken to implement these goals as best as possible will transformatively improve SPEX's advanced payload capabilities.
+
+\subsection{Milestones}
+Development and fabrication milestones are listed in \autoref{milestones-table}.
+\begin{table}[!ht]
+  \centering
+  \caption{Milestones to design and build HAB5}
+
+\begin{minipage}{.45\textwidth}
+  \noindent
+  \begin{tabular}{@{}ll@{}}
+
+  \label{milestones-table}
+    \textbf{Task} & \textbf{Complete Date} \\
+    \midrule
+    Review and Write Up of Current Systems & September 2nd Week \\
+    Mechanical Constraints for Development & September 2nd Week \\ 
+    Sensor Acquisition, Readout Config & October 2nd Week \\
+    Power System and Communications Design & October 2nd Week \\
+    Mechanical Design \footnote{Mechanical deadlines assume rebuilding the box to the HAB4 specification. This work is truncated if the HAB4 structure is not damaged from previous flights.} & October 2nd Week \\
+    Camera Setup and Software & October 2nd Week \\
+    Power System Assembly & October 4th Week \\
+    Mechanical Construction & October 4th Week \\
+    Image Processing Implementation, Testing & December \\
+    APRS Testing and Verification\footnote{Assumes the HAB4 power board is supplying power and APRS to HAB5 avionics boards.} & December \\
+    Testing of All Subsystems & February 2nd Week \\
+    Image Processing Tuning & February 2nd Week \\
+    Assembly and Testing, incl. Environmental & February 4th Week \\
+    Launch & Imagine RIT \\
+  \end{tabular}
+\end{minipage}
+\end{table}
+
+\section{Externalities}
+\subsection{Prerequisite Skills}
```

Comment:
An estimated minimum number of people required for project success would be nice to have. Also how much mechanical fabrication and analysis is expected? What would be the maximum number of people the project could support?

Reply:
I believe the questions here were leading ones to tease out more detail in the document. Given a team of members that show up to every meeting but are not working to the bone, describe what size team would be ideal so that everyone has meaningful work, is working efficiently, and all the milestones are met according to the timeline you lay out above.